Dockerfile

```diff
@@ -55,6 +55,8 @@ RUN cp -r /tmp/usr/bin/clamdscan \
        /tmp/usr/bin/freshclam \
        /tmp/usr/lib64/* \
        /opt/app/bin/
+       
+RUN cp /usr/lib64/libpcre.so.1.2.0 /opt/app/bin/libpcre.so.1       
```

Comment:
I think this file should have been in the `/tmp/usr/lib64` directory as part of the build?